product_docs/docs/pgd_cli/1/pgd_show-nodes.mdx

```diff
@@ -2,21 +2,21 @@
 title: show-nodes
 ---
 
-Shows all nodes in the BDR cluster and their summary.
+Shows all nodes in the EDB Postgres Distributed cluster and their summary.
 
 ### Synopsis
 
-Shows all nodes in the BDR cluster and their summary such as name, node id,
-group, and current/target state etc.
+Shows all nodes in the EDB Postgres Distributed cluster and their summary such
```

Comment:
We try to avoid Latin abbreviations. Can you change this in the source to: Shows all nodes in the EDB Postgres Distributed cluster and their summary including name, node id, group, and current/target state.